app/src/main/java/it/quip/android/util/MockUtils.java

```diff
@@ -58,6 +70,10 @@ public static Circle circleWithNameAndMembers(String name, List<User> members) {
         return circle;
     }
 
+    public static Quip quipWithText(String text) {
+        return new Quip(randomId(), text, randomUser(), randomUser(), randomCircle(), randomTimestamp());
```

Comment:
Yeah this was me being lazy and not wanting to write out JSON string. I'll remove it shortly.